src/patternfly/upgrade-examples/AlertUpgradeExamples/examples/alert-upgrade-example-pf3-notification.hbs

```diff
@@ -24,8 +24,8 @@
   </div>
   <br>
   <div class="toast-pf alert alert-info">
-    <div class="dropdown pull-right dropdown-kebab-pf">
-      <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+    <div class="dropdown pull-right dropdown-kebab-pf" aria-label="dropdown toggle">
+      <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true" aria-label="dropdown toggle">
```

Comment:
Could you change this to `aria-label="Actions"`?

I'm on the fence about updating the PF3 examples on this page. There will be cases where we cannot fix violations that get flagged. But I'm not opposed to following best practice where possible.

Reply:
`aria-label` changed